src/app/pages/IdxPage/index.test.jsx

```diff
@@ -44,3 +45,38 @@ describe('IdxPage', () => {
     });
   });
 });
+
+describe('IdxPage - Ukrainian', () => {
+  beforeEach(async () => {
+    fetchMock.restore();
+
+    await act(async () => {
+      container = render(
+        <IdxPageWithContext
+          service="ukrainian"
+          pageData={ukraineInRussianIdxData}
+        />,
+      ).container;
+    });
+  });
+
+  describe('Snapshots', () => {
+    it('should render a persian idx page correctly with most read and radio schedule', async () => {
+      expect(container).toMatchSnapshot();
+    });
+  });
+
+  describe('Assertions', () => {
+    it('should render idx page sections', async () => {
+      const sections = container.querySelectorAll('section');
+      expect(sections).toHaveLength(4);
```

Comment:
The number of sections might change and that would still be valid. It'd be better to check that the number of sections is greater than 1.